content/systems/filecoin_blockchain/chainsync/_index.md

```diff
@@ -178,177 +80,24 @@ of the altered protocol.
 
-### Notes on changing `TargetHeads` while syncing
+For more details on Peer Exchange please refer to the [GossipSub specification](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub).
 
-- `TargetHeads` is changing, as `ChainSync` must be aware of the best heads at any time. reorgs happen, and our first set of peers could've been bad, we keep discovering others.
-  - Hello protocol is good, but it's polling. Unless node is constantly polllng, won't see all the heads.
-  - `BlockPubsub` gives us the realtime view into what's actually going on.
-  - Weight can also be close between 2+ possible chains (long-forked), and `ChainSync` must select the right one (which, we may not be able to distinguish until validating all the way)
-- fetching + validation are strictly faster per round on average than blocks produced/block time (if they're not, will always fall behind), so we definitely catch up eventually (and even quickly). The last couple rounds can be close ("almost got it, almost got it, there").
+### Explicit Peering Agreements
 
-### General notes on fetching Blocks
+With explicit peering agreements, the operators must specify a list of peers which nodes should connect to when joining. The protocol must have options available for these to be specified. For every explicit peer, the router must establish and maintain a bidirectional (reciprocal) connection.
 
-- `ChainSync` selects and maintains a set of the most likely heads to be correct from among those received
-  via `BlockPubsub`. As more blocks are received, the set of `TargetHeads` is reevaluated.
-- `ChainSync` fetches `Blocks`, `Messages`, and `StateTree` through the `Graphsync` protocol.
-- `ChainSync` maintains sets of `Blocks/Tipsets` in `Graphs` (see `ChainSync.id`)
-- `ChainSync` gathers a list of `TargetHeads` from `BlockPubsub`, sorted by likelihood of being the best chain (see below).
-- `ChainSync` makes requests for chains of `BlockHeaders` to close gaps between  `TargetHeads`
-- `ChainSync` forms partial unvalidated chains of `BlockHeaders`, from those received via `BlockPubsub`, and those requested via `Graphsync`.
-- `ChainSync` attempts to form fully connected chains of `BlockHeaders`, parting from `StateTree`, toward observed `Heads`
-- `ChainSync` minimizes resource expenditures to fetch and validate blocks, to protect against DOS attack vectors.
-  `ChainSync` employs **Progressive Block Validation**, validating different facets at different stages of syncing.
-- `ChainSync` delays syncing `Messages` until they are needed. Much of the structure of the partial chains can
-  be checked and used to make syncing decisions without fetching the `Messages`.
-
-### Progressive Block Validation
+## Progressive Block Validation
```

Comment:
We should merge https://github.com/filecoin-project/lotus/blob/master/documentation/en/block-validation.md here if possible, or at least extract the checks required by the spec and leave there just implementation specific ones.